### DISCUSSION TITLE
提几点体验上的建议和想法：

### DISCUSSION THREAD

**Discussion by ker-j** [qier222/YesPlayMusic#1751] [Ideas]
1.为什么不做本地音乐播放器呢？目前我认为这一款是除mac音乐外最优秀的音乐播放器了，有了本地功能简直绝配
2.歌词界面还可以完善，尤其是歌词界面只有黑色背景，如有白色背景则更棒
3.歌词背景如果可以模糊+动态渐变，类似官方音乐那种歌词效果就更棒了

**Comment by pan93412** [comment]
1. 這一版要加入本地播放比較麻煩；未來的重寫版本有計劃加入這方面的功能。
2. 歌詞頁面是以歌曲封面選色的。不能選色很有可能是 bug，可以在 https://github.com/qier222/YesPlayMusic/issues/1718 附議。

![](https://user-images.githubusercontent.com/28441561/177032895-52cbe034-ffce-4233-a695-067cfcaf0d14.png)


![](https://user-images.githubusercontent.com/28441561/177032913-5b3a3705-f341-4f0d-861b-2c40d95d2627.png)

3. 見上
